src/components/Swapping.tsx

```diff
@@ -17,10 +17,10 @@ import paraswapIcon from '../assets/icons/paraswap.png'
 import walletIcon from '../assets/wallet.png'
 import { oneInch } from '../services/1Inch'
 import { AnySwapExecutionManager } from '../services/anyswap.execute'
-import { getBalancesForWallet } from '../services/balanceService'
 import { CbridgeExecutionManager } from '../services/cbridge.execute'
 import { HopExecutionManager } from '../services/hop.execute'
 import { HorizonExecutionManager } from '../services/horizon.execute'
+import Lifi from '../services/LIFI/Lifi'
```

Comment:
The SDK does not have the config file the normal balance service has. I understand, that it is not necessary for the SDK to allow for maximum freedom.
However, blocking malicious tokens on the user-facing site seems like a nice feature.